reference-implementation/test/readable-stream-reader.js

```diff
@@ -155,7 +155,8 @@ test('Multiple readers can access the stream in sequence', t => {
   reader2.read().then(r => t.deepEqual(r, { value: 'b', done: false }, 'reading the second chunk from reader2 works'));
   reader2.releaseLock();
 
-  t.end();
+  // Let promises run.
+  setTimeout(() => t.end(), 50);
```

Comment:
Good catch! It would be better to use t.plan(2) at the top of the test though, I think.